### PR TITLE
add facebook OAUTH

### PR DESCRIPTION
**Pull request by eddyspaghette**
added fb login and google login OAUTH

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -13,4 +13,8 @@
     <string name="menu_home">Home</string>
     <string name="menu_treasure">Treasure</string>
     <string name="menu_feed">Feed</string>
+
+
+    <string name="facebook_application_id">1355344324874942</string>
+    <string name="facebook_client_token">a488ec6c4c19e8b4198614eaff47293d</string>
```
Comment (akn0717) [inline app/src/main/res/values/strings.xml:19]: is that the Facebook token linked to your FB account?

Comment (eddyspaghette) [reply app/src/main/res/values/strings.xml:19]: nope its the application id and client token required to use the facebook api

Comment (akn0717) [reply app/src/main/res/values/strings.xml:19]: ok!

Comment (raizapw19) [inline app/src/main/res/values/strings.xml:19]: Do we want to include this here?

**Review by akn0717 — APPROVED** [review]
LGTM!

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (eddyspaghette) [reply app/src/main/res/values/strings.xml:19]: safe to use from this post (links to official documentation) : https://stackoverflow.com/questions/72121612/is-the-facebook-client-id-safe-to-commit-to-a-public-repository

Comment (eddyspaghette) [reply app/src/main/res/values/strings.xml:19]: safe to use from this post (links to official documentation) : https://stackoverflow.com/questions/72121612/is-the-facebook-client-id-safe-to-commit-to-a-public-repository

Comment (raizapw19) [reply app/src/main/res/values/strings.xml:19]: nvm, answered above

**Pull request merged**